protocols/s3/synology.md

```diff
@@ -0,0 +1,16 @@
+Synology C2 Object Storage
+====
+
+> [C2 Object Storage](https://c2.synology.com/en-global/object-storage/overview) is a powerful and flexible solution designed to help businesses manage and store massive amounts of data in a cost-effective way. 
+
+## Connecting
+
```

Comment:
Add a note _All connection profiles are available through the Preferences → Profiles tab._ as on other pages [^1]. 

[^1]: https://docs.cyberduck.io/protocols/s3/digitalocean/#connecting

Reply:
done